### DISCUSSION TITLE
erro when starting myst : request to https://api.mystmd.org/templates/site/myst/book-theme failed, reason: getaddrinfo ENOTFOUND api.mystmd.org

### DISCUSSION THREAD

**Discussion by WiG96** [executablebooks/MyST-Parser#998] [Q&A]
i'm just starting with myST following this tutorial https://mystmd.org/guide/website-templates. I run into this error `request to https://api.mystmd.org/templates/site/myst/book-theme failed, reason: getaddrinfo ENOTFOUND api.mystmd.org`
![image](https://github.com/user-attachments/assets/2c6cb012-44b7-4d8b-a95d-8ea563059ca9)

myst.yml file : 

![image](https://github.com/user-attachments/assets/d6e40dca-4d6d-4b75-80c2-b5fcdf2f6b28)

**Comment by chrisjsewell** [comment]
That's not myst-parser

**Comment by agoose77** [comment]
Hi @WiG96, as @chrisjsewell points out, this repository is the home of a different project to mystmd.

The `mystmd` repository lives over at https://github.com/jupyter-book/mystmd! I'll transfer the issue.
